yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,512 @@
+
+<center> "My guess is <span style ="background-color: green">ten</span>."</center>
+
+# Define the Second Intent Handler and Endpoint Function
+
+## Define the Second Intent Handler for *makeGuessIntent* (Part 1)
+
+Now that we have defined the Interaction Mode of our new **makeGuessIntent**, we need to define the “brain” of the Alexa intent. Our skill can now recognize when we give it our guess, but it doesn’t know what to do or how to handle it. To make the response, we need to define the <span class = "control">when makeGuessIntent spoken </span>block to tell Alexa what to do when you trigger the **“makeGuessIntent”**. To begin, let’s go back to the Blocks view.
+
+![Blocks View](../images/number_guessing_game/image7.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Here, should see our **makeGuessIntent** as its own Component darwer to the left. Click on that to open it up and drag out the <span class = "control">when makeGuessIntent.spoken </span> block into your workspace.
+
+![makeGuessIntent block](../images/number_guessing_game/image23.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Whenever our Skill hears the user trigger the **makeGuessIntent**, this block will run anything inside of it.
+
+Let's first have our Skill repeat our guess and say: "You guessed (value of myGuessSlot)". Similar to before, we need to use the **say** block from the **Voice** drawer and the <span class = "text">join</span> block from the **Text** drawer. Try implementing everything except (myGuessSlot) yourself! Click the hint when you're finished to see what your blocks should look like.
+
+<hint markdown="block" title="Answer!">
+![Answer](../images/number_guessing_game/image49.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+</hint>
+
+In order to get the value of our slot **myGuessSlot**, click the **myGuessSlot** drawer on the lower left of the **Components** block, and drag out the **myGuessSlot.value** block into the <span class = "text">join</span> block.
+
+![Slot Components](../images/number_guessing_game/image51.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+![makeGuessIntent Spoken](../images/number_guessing_game/image22.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+## Define the Intent Handler for *makeGuessIntent* (Part 2)
+Now we want our **makeGuessIntent** to respond differently based on our **myGuessSlot** value. **IF** our guess is greater than our **secret** variable, **THEN** Alexa will **ask**: “(myGuessSlot) is too big. Guess again”. The user will then have another opportunity to guess, which will create a loop until the user guesses correctly. **IF** our guess is less than our **secret** variable, **THEN** Alexa will **ask**: “(myGuessSlot) is too small. Guess again”, which will once again create a loop. **IF** our guess is accurate, **THEN** Alexa will say: “<myGuessSlot> is correct. Congratulations!"
```

Comment:
Similar issues as before not sure what is the intended formatting for (myGuessSlot).
Also check here and everywhere when you use < myGuessSlot > nothing shows up.